openfisca_core/scripts/openfisca_command.py

```diff
@@ -41,6 +41,7 @@ def build_test_parser(parser):
         parser.add_argument('-v', '--verbose', action = 'store_true', default = False, help = "increase output verbosity")
         parser.add_argument('-o', '--only-variables', nargs = '*', default = None, help = "variables to test. If specified, only test the given variables.")
         parser.add_argument('-i', '--ignore-variables', nargs = '*', default = None, help = "variables to ignore. If specified, do not test the given variables.")
+        parser.add_argument('-d', '--depth_max', type = int, default = None, help = "Maximal depth of the log. If not specified, display all the computation")
```

Comment:
```suggestion
        parser.add_argument('-l', '--max-log-depth', type = int, default = None, help = "Maximal log depth. If specified, cap log verbosity to the provided calculation depth.")
```

Reply:
@benjello Makes sense?

Reply:
@maukoquiroga : aggregate and calculation depth are orthogonal.
Depth is about showing the value of calculation when you have variable A depending on B depending on C etc.
You may wwant to stop at B (`max-depth = 2`).

Aggregate is about showing `mean`/`max`/`min` of the variables A or B o C on your whole population.

Reply:
Being orthogonal means they should play well together then.

Reply:
>  This flag is incompatible with --aggregate."
This is thus not relevant.